packages/ui-chart-components/src/components/Reference/ReferenceLines.tsx

```diff
@@ -1,91 +1,50 @@
 /*
  * Tupaia
- * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
  */
 
 import React from 'react';
 import { ReferenceLine } from 'recharts';
 import { formatDataValueByType } from '@tupaia/utils';
-import { ChartConfigObject, ChartType } from '@tupaia/types';
+import { ChartType } from '@tupaia/types';
 import { TUPAIA_ORANGE } from '../../constants';
-import { CartesianChartViewContent, ViewContent } from '../../types';
+import { CartesianChartViewContent } from '../../types';
 import { ReferenceLabel } from './ReferenceLabel';
 
-const ReferenceLineLabel = ({
-  referenceLineLabel,
-  isExporting,
-}: {
-  referenceLineLabel?: string;
-  isExporting?: boolean;
-}) => {
-  if (referenceLineLabel === undefined) return undefined;
-  return <ReferenceLabel value={referenceLineLabel} fill={isExporting ? '#000000' : '#ffffff'} />;
-};
-
-enum Y_AXIS_IDS {
-  left = 0,
-  right = 1,
-}
-
-const DEFAULT_Y_AXIS = {
-  id: Y_AXIS_IDS.left,
-  orientation: 'left',
-  yAxisDomain: {
-    min: { type: 'number', value: 0 },
-    max: { type: 'string', value: 'auto' },
-  },
-};
-
-const orientationToYAxisId = (orientation?: 'left' | 'right'): number =>
-  (orientation && Y_AXIS_IDS[orientation]) || DEFAULT_Y_AXIS.id;
-
-function hasReferenceValue(chartConfig?: ChartConfigObject) {
-  return chartConfig && chartConfig?.referenceValue !== undefined;
-}
-
-const ValueReferenceLine = ({
-  viewContent,
-  isExporting,
-}: {
+interface ReferenceLineProps {
   viewContent: CartesianChartViewContent;
   isExporting?: boolean;
-}) => {
-  const { chartConfig } = viewContent;
-
-  if (!chartConfig || !hasReferenceValue(chartConfig)) {
-    return [];
-  }
+  isEnlarged?: boolean;
+}
 
-  const referenceLines = Object.entries(chartConfig)
-    .filter(([, { referenceValue }]) => referenceValue)
-    .map(([dataKey, { referenceValue, yAxisOrientation, referenceLabel }]) => ({
-      key: `reference_line_${dataKey}`, // Use prefix to distinguish from curve key
-      y: referenceValue,
-      yAxisId: orientationToYAxisId(yAxisOrientation),
-      referenceLineLabel: referenceLabel,
-    }));
+const ValueReferenceLine = ({ viewContent, isExporting }: ReferenceLineProps) => {
+  const hasReferenceLine = !!viewContent.presentationOptions?.referenceLines?.targetLine;
+  if (!hasReferenceLine) return null;
```

Comment:
I had hoped to break this into

```ts
const hasReferenceLine = !!viewContent.presentationOptions?.referenceLines?.targetLine;
if (!hasReferenceLine) return null;
```

but then TypeScript thinks `referenceLines` could be undefined in the next statement. I just put the ugly chain inline because it’s the only usage of `targetLine`, so wasn’t really worth a dedicated type guard